docs/jsonapi-spec.md

```diff
@@ -536,14 +536,93 @@ Creates a new collection in the current namespace.
 ```json
 {
   "createCollection": {
-    "name": "purchase"
+    "name": "some_collection",
+    "options" : {
+      "vector" : {
+      "dimension" : 5,
+      "metric" : "cosine"
+      }
+    }
   }
 }
 ```
 
 #### createCollection Command Options
 
-The `createCollection` command does not support any options.
+The `createCollection` command supports the following options.
+
+| Request Elements | Description                                          |
+| ---------------- | -----------------------------------------------------|
+| `vector`         | Used to define a vector-enabled collection.          |
+| `dimension`      | The size or dimension of the vector.                 |
+| `metric`         | One of: `dot_notation`, `euclidean`, or `cosine`.    |
```

Comment:
Shouldn't `dot_notation` be `dot_product`?

Reply:
Thanks. Done in most recent commit.